README.md

```diff
@@ -44,36 +42,33 @@ Set Up PostgreSQL with Docker (docker-compose.yml provided)
 
 ### 4. Environment Variables
 
-in backend root:
+In the `backend` directory:
 
 ```sh
-BLOCKSTREAM_API_URL=https://blockstream.info/testnet/api #test-api
-DATABASE_URL=postgres://<<your-user>>>>:<<your-passward>>>>@localhost:5432/<<bitsnark or other name>>
```

Comment:
I think something bad happened here with the angle brackets.

Reply:
Also, "or other name" is confusing. I assume that this is the name of the database or schema, and I think it's perfectly okay to just hard code it for now.

Reply:
Also, is this really enough? No need to export the variables?

Reply:
Its all suppose to go into an .env file. I wan't clear enough...